### ISSUE TITLE
InstallGenerator::DashboardManifest (NameError)

### ISSUE THREAD

**Issue by Qcoin** [thoughtbot/administrate#453]
Hello everyone, 
I have a problem with administrate, if I install i have this:
```
boboss@boboss:~/projet/my_app$ rails generate administrate:install
Running via Spring preloader in process 9965
      create  app/controllers/admin/application_controller.rb
       route  namespace :admin do
    DashboardManifest::DASHBOARDS.each do |dashboard_resource|
      resources dashboard_resource
    end

    root controller: DashboardManifest::ROOT_DASHBOARD, action: :index
  end
      create  app/dashboards/dashboard_manifest.rb
/home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:44:in `manifest': uninitialized constant Administrate::Generators::InstallGenerator::DashboardManifest (NameError)
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:36:in `dashboard_resources'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:32:in `singular_dashboard_resources'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:24:in `run_dashboard_generators'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `block in invoke_all'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `each'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `map'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `invoke_all'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/group.rb:232:in `dispatch'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/generators.rb:157:in `invoke'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/generate.rb:13:in `<top (required)>'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:123:in `require_command!'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:130:in `generate_or_destroy'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:50:in `generate'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands.rb:17:in `<top (required)>'
	from /home/boboss/projet/lucenay/bin/rails:9:in `<top (required)>'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from -e:1:in `<main>'
```

I add this issues, i rebundle
#450 and #435

I have this result:
```
boboss@boboss:~/projet/my_app$ DISABLE_SPRING=1 rails generate administrate:install
      create  app/controllers/admin/application_controller.rb
       route  namespace :admin do
    DashboardManifest::DASHBOARDS.each do |dashboard_resource|
      resources dashboard_resource
    end

    root controller: DashboardManifest::ROOT_DASHBOARD, action: :index
  end
      create  app/dashboards/dashboard_manifest.rb
      create  app/dashboards/demarch_dashboard.rb
/home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/generators/administrate/dashboard/dashboard_generator.rb:107:in `const_get': uninitialized constant Demarch (NameError)
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/generators/administrate/dashboard/dashboard_generator.rb:107:in `klass'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/generators/administrate/dashboard/dashboard_generator.rb:47:in `attributes'
	from (erb):11:in `template'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/2.2.0/erb.rb:863:in `eval'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/2.2.0/erb.rb:863:in `result'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/file_manipulation.rb:116:in `block in template'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:53:in `call'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:53:in `render'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:62:in `block (2 levels) in invoke!'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:62:in `open'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:62:in `block in invoke!'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/empty_directory.rb:116:in `call'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/empty_directory.rb:116:in `invoke_with_conflict_check'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:60:in `invoke!'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions.rb:94:in `action'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:25:in `create_file'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/file_manipulation.rb:115:in `template'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/generators/named_base.rb:26:in `block in template'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/generators/named_base.rb:60:in `inside_template'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/generators/named_base.rb:25:in `template'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/generators/administrate/dashboard/dashboard_generator.rb:30:in `create_dashboard_definition'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `block in invoke_all'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `each'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `map'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `invoke_all'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/group.rb:232:in `dispatch'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/generators.rb:157:in `invoke'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/administrate/generator_helpers.rb:4:in `call_generator'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:25:in `block in run_dashboard_generators'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:24:in `each'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:24:in `run_dashboard_generators'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `block in invoke_all'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `each'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `map'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `invoke_all'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/group.rb:232:in `dispatch'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/generators.rb:157:in `invoke'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/generate.rb:13:in `<top (required)>'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:123:in `require_command!'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:130:in `generate_or_destroy'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:50:in `generate'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /home/boboss/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:9:in `require'
	from bin/rails:9:in `<main>'
```

Thanks for your help.

**Comment by steffenix** [comment]
Isn't it the same as:
https://github.com/thoughtbot/administrate/issues/448
https://github.com/thoughtbot/administrate/issues/431
and there is a PR for that:
https://github.com/thoughtbot/administrate/pull/435

**Comment by Qcoin** [comment]
Hello,
yes, you're right it the same but not solve my problem.
I tried version 1.2 without any results.

**Comment by graysonwright** [comment]
I created #464 to discuss potential solutions to this. Please join in the conversation there!

**Comment by graysonwright** [comment]
Closed by #464, which will go out in `0.2.0`.
